pkg/api/types.go

```diff
@@ -728,3 +742,17 @@ const (
 	// NodeRepoType represents the repository type NodeJS.
 	NodeRepoType string = "NodeJS"
 )
+
+// NotificationContent contains some pipeline record infomation.
+type NotificationContent struct {
+	ProjectName  string    `yaml:"projectName,omitempty" json:"projectName,omitempty" `
+	PipelineName string    `yaml:"pipelineName,omitempty" json:"pipelineName,omitempty" `
+	RecordName   string    `yaml:"recordName,omitempty" json:"recordName,omitempty" `
+	RecordID     string    `yaml:"recordID,omitempty" json:"recordID,omitempty" `
+	Trigger      string    `bson:"trigger,omitempty" json:"trigger,omitempty" description:"trigger of the pipeline record"`
+	Status       Status    `bson:"status,omitempty" json:"status,omitempty" description:"status of the pipeline record"`
+	ErrorMessage string    `bson:"errorMessage,omitempty" json:"errorMessage,omitempty" description:"error message for the pipeline failure"`
+	StartTime    time.Time `bson:"startTime,omitempty" json:"startTime,omitempty" description:"start time of the pipeline record"`
+	EndTime      time.Time `bson:"endTime,omitempty" json:"endTime,omitempty" description:"end time of the pipeline record"`
+	TimeCost     float64   `bson:"timeCost,omitempty" json:"timeCost,omitempty" description:"time cost, unit(second), endTime - startTime"`
```

Comment:
`TimeCost` is not needed to defined, as it can be got by `StartTime` and `EndTime`